src/index.ts

```diff
@@ -32,6 +33,8 @@ export {
   usePageActiveTrigger
 };
 
+export const plugin = { getFlushedPayload };
```

Comment:
hi hi Conner, super thanks for point this out. I don't know to be honest and I need to find a way to verify too. In order to not blocking the release, I'll change this top-level export for now and verify it later on. Does this sound good to you? :)

Reply:
According to this [StackOverflow answer](https://stackoverflow.com/a/56501534/4620154), Webpack 4 doesn't handle nested exports but Webpack 5 does. Many of our applications still use Webpack 4, so a top-level export is preferable. Thanks for updating the code! 

If we change this in the future, this would be a breaking change since it changes the public API of the module.

Reply:
Ahhh got it. thanks for sharing!! 👍 👍 👍